### PR TITLE
Use hash_equals() to compare hashes

### PR DESCRIPTION
**Pull request by NiklasBr**
Just a small change to use https://www.php.net/manual/en/function.hash-equals.php

### CODE REVIEW

**Comment by NiklasBr** [comment]
**_I have read the CLA Document and I hereby sign the CLA_**

File: src/DataSource/Interpreter/DeltaChecker/DeltaChecker.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public function hasChanged(string $configName, $idDataIndex, array $data): bool
         $oldHash = $this->getCurrentHash($configName, $id);
 
         $newHash = md5(json_encode($data));
-        if ($oldHash !== $newHash) {
+        if (hash_equals($oldHash, $newHash)) {
```
Comment (blankse) [inline src/DataSource/Interpreter/DeltaChecker/DeltaChecker.php]:
```suggestion
        if (!hash_equals($oldHash, $newHash)) {
```

**Comment by mcop1** [comment]
@NiklasBr Thank you for the pr. Could you please try and merge 1.7 into your pr again? This should fix/deactivate the failing test.

**Comment by mcop1** [comment]
Thanks :)

**Pull request merged**